### PR TITLE
Hide goal description diff in history under cut

### PR DESCRIPTION
**Pull request by Troff8**
## PR includes

- [x] Feature

## Related issues

Resolve [#1238 ](https://github.com/taskany-inc/issues/issues/1238)

## QA Instructions, Screenshots, Recordings
  

https://github.com/taskany-inc/issues/assets/87986652/2821d7b3-f106-4e05-8924-f8e4bf46d670

https://github.com/taskany-inc/issues/assets/87986652/160b8595-1341-4f32-b4e8-600fc01079da

### CODE REVIEW

File: src/components/HistoryRecord/HistoryRecord.tsx [hunk 1]
```diff
@@ -149,11 +153,16 @@ export const HistoryRecord: React.FC<HistoryRecordProps> = ({ author, subject, a
     }, []);
 
     const [actionText, setActionText] = useState(action);
+    const [viewDestiption, setViewDescription] = useState(false);
+
+    const handlerViewDescription = useCallback(() => {
+        setViewDescription(!viewDestiption);
+    }, [viewDestiption]);
```
Comment (9teen90nine) [inline src/components/HistoryRecord/HistoryRecord.tsx:160]: unnecessary useCallback

**Review by 9teen90nine — CHANGES_REQUESTED** [review]

File: src/components/HistoryRecord/HistoryRecord.tsx (hunk 1 above)
Comment (Troff8) [reply src/components/HistoryRecord/HistoryRecord.tsx]: ok 💪🏻

**Review by 9teen90nine — APPROVED** [review]

**Review by awinogradov — CHANGES_REQUESTED** [review]
Do not change icon color

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/components/HistoryRecord/HistoryRecord.tsx [hunk 2]
```diff
@@ -164,9 +174,11 @@ export const HistoryRecord: React.FC<HistoryRecordProps> = ({ author, subject, a
                             {translates[actionText]} {translates[subject]}
                         </Text>
                         {children}
-                        <Text size="xs">
-                            <RelativeTime date={createdAt} />
-                        </Text>
+                        {action !== 'change' && (
```
[the remark this answers is not in the document]
Comment (Troff8) [reply src/components/HistoryRecord/HistoryRecord.tsx:177]: {action:'change'} only appears when the title/description changes. For the title and description component, I added a <RelativeTime> . This was necessary in order not to add another action

File: src/components/HistoryRecord/HistoryRecord.tsx [hunk 3]
```diff
@@ -238,18 +250,58 @@ export const HistoryRecordProject: React.FC<HistoryChangeProps<Project>> = ({ fr
...
+                from={
+                    from ? (
+                        <StyledText size="xs" strike>
+                            {from}
+                        </StyledText>
+                    ) : null
+                }
+                to={to ? <StyledText size="xs">{to}</StyledText> : null}
+            />
+            <Text size="xs">
```
[the remark this answers is not in the document]
Comment (Troff8) [reply src/components/HistoryRecord/HistoryRecord.tsx:299]: I added <RelativeTime> in the right place - before the button.

**Review by awinogradov — CHANGES_REQUESTED** [review]
image

**Review by awinogradov — CHANGES_REQUESTED** [review]
Still...

image

**Review by awinogradov — APPROVED** [review]

**Pull request merged**